src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -41,14 +42,23 @@
 #include "utils/palloc.h"
 #include "utils/relcache.h"
 
+/* expression tree walker context for rewriting row references */
```

Comment:
We always put `struct`s in header files. Yes, even ones that are "private". It's not the most popular decision, but for consistency this should be moved.